Formula/newrelic-cli.rb

```diff
@@ -0,0 +1,28 @@
+class NewrelicCli < Formula
+  desc "The New Relic Command-line Interface"
+  homepage "https://github.com/newrelic/newrelic-cli"
+  url "https://github.com/newrelic/newrelic-cli/archive/v0.2.3.tar.gz"
+  sha256 "f09d7e1705731578da79f0d9c204a6875edbd1d78cc6e69bd9f7d00dd9919986"
+  head "https://github.com/newrelic/newrelic-cli.git"
+
+  depends_on "go" => :build
+
+  def install
+    ENV["GOPATH"] = buildpath
+    ENV.prepend_create_path "PATH", buildpath / "bin"
+    ENV["PROJECT_VER"] = version
+
+    dir = buildpath / "src/github.com/newrelic/newrelic-cli"
+    dir.install buildpath.children - [buildpath / ".brew_home"]
+
+    cd dir do
+      system "make", "compile-only"
+
+      bin.install "bin/darwin/newrelic"
+    end
```

Comment:
Do you support Go modules? If so, you could simplify a lot of this (example formula which moved to use Go modules: https://github.com/Homebrew/homebrew-core/commit/8b7ebcbbcbf5fa60677e1ae8d7b59324bd83a2ce).

Reply:
We do support go modules, I wasn't aware that we could remove all of that.  Cleaner now.